app/models/User.php

```diff
@@ -29,4 +29,9 @@ public function certificateValid($certificate = null)
 
 		return true;
 	}
+
+	public function isAdmin()
+	{
+		return in_array(strtolower($this->username), array_map('trim', explode(',', $_ENV['PHRAGILE_ADMINS'])));
```

Comment:
A bit too much complexity on one line IMO. I'd indent like

```php
return in_array(
    strtolower($this->username),
    array_map('trim', explode(',', $_ENV['PHRAGILE_ADMINS']))
);
```

Or whatever the equivalent with this CS is.